managed/CounterStrikeSharp.API/Modules/Entities/Constants/SteamAccountUniverse.cs

```diff
@@ -0,0 +1,32 @@
+namespace CounterStrikeSharp.API.Modules.Entities.Constants;
+
+/// <summary>
+/// Represents the universe of a Steam account.
+/// </summary>
+public enum SteamAccountUniverse
+{
+    /// <summary>
+    /// Invalid universe.
+    /// </summary>
+    Invalid = 0,
```

Comment:
I would suggest going the middle ground and go with "Unspecified" to not loose some of that. Since, "Individual" does not indicate above mentioned at all.